lib/open_api/docs.rb

```diff
@@ -7,7 +7,11 @@ def initialize(glob)
 
     def load_file(file)
       yaml = YAML.load_file(file)
-      doc  = DocV2.new(yaml) if yaml["swagger"] == "2.0"
+      doc = if yaml["swagger"] == "2.0"
+              DocV2.new(yaml)
+            elsif yaml["openapi"] =~ /3*/
+              DocV3.new(yaml)
+            end
```

Comment:
Not sure I follow the need for this if we don't have v2 anymore.  Additionally, it's not a yaml anymore, right, so is the variable name wrong?

Reply:
yaml really just a hash :) this code would change of course if we drop DocV2 altogether.